src/server.rs

```diff
@@ -122,6 +124,12 @@ impl<T: Read + Write + ScmSocket> ClientConnection<T> {
                     return Ok(vec![]);
                 }
                 Err(ConnectionError::StreamReadError(inner)) => {
+                    #[cfg(target_os = "linux")]
+                    if inner.errno() == libc::EAGAIN {
+                        sleep(Duration::from_micros(200));
```

Comment:
200ms seems a little long, how about 10?

Reply:
It's microseconds(us) rather milliseconds (ms)

Reply:
Do you mean 10us?

Reply:
Changed to 20us